extensions/redis-client/deployment/src/main/java/io/quarkus/redis/client/deployment/DevServicesRedisProcessor.java

```diff
@@ -42,7 +42,7 @@
 @BuildSteps(onlyIfNot = IsNormal.class, onlyIf = { GlobalDevServicesConfig.Enabled.class })
 public class DevServicesRedisProcessor {
     private static final Logger log = Logger.getLogger(DevServicesRedisProcessor.class);
-    private static final String REDIS_6_ALPINE = "docker.io/redis:6-alpine";
+    private static final String REDIS_6_ALPINE = "docker.io/redis:7-alpine";
```

Comment:
Should you rename the constant to `REDIS_ALPINE` instead?

Reply:
oh yes!